sympy/ntheory/continued_fraction.py

```diff
@@ -154,7 +155,7 @@ def untillist(cf):
 
     if period:
         y = Dummy('y')
-        solns = solve(continued_fraction_reduce(period + [y]) - y, y)
+        solns = list(solveset(continued_fraction_reduce(period + [y]) - y, y))
```

Comment:
Sorry about all the spam. I didn't look closely at the merged ones.

Thanks for fixing these and leaving any tests for solve in place.